### PR TITLE
Makefile

### PR DESCRIPTION
**Pull request by gtramontina**
### Summary
- Move the `install` target up so it is the default, when you run `make`
- Set all targets as `PHONY`.

### CODE REVIEW

**Comment by filipesperandio** [comment]
Re: Install - I was going in this direction till I realised ubuntu does not come with `make` by default, so no reason to have install as default since we will have to call `./install.sh`  anyways.

I don't understand PHONY, what is this?

File: README.md [hunk 1]
```diff
@@ -3,5 +3,5 @@
 1. Clone this repo
 2. Run:
 ```shell
-./install.sh
+make
```
Comment (filipesperandio) [inline README.md]: No cannot do it. :(

**Comment by fgrehm** [comment]
@filipesperandio re PHONY: http://stackoverflow.com/a/2145605

**Comment by filipesperandio** [comment]
Got it.

**Pull request merged**